Source/ACE/Entity/Creature.cs

```diff
@@ -301,6 +301,47 @@ public void HandleUnloadMissileAmmo(CombatMode oldCombatMode)
         }
 
         /// <summary>
+        /// GetWilded Items
+        /// </summary>
+        /// <param name="objectGuid"></param>
+        /// <returns>WorldObject</returns>
+        public virtual WorldObject GetWieldedItem(ObjectGuid objectGuid)
+        {
+            // check wielded objects
+            WorldObject inventoryItem;
+            if (WieldedObjects.ContainsKey(objectGuid))
+            {
+                if (WieldedObjects.TryGetValue(objectGuid, out inventoryItem))
+                    return inventoryItem;
+            }
+            return null;
+        }
+
+        public uint GetCreatureInventoryFreePack()
+        {
+            return GetFreePackLocation();
+        }
+
+        /// <summary>
+        /// Returns a inventory item by its objectGuid from any creature pack.
+        /// </summary>
+        /// <param name="objectGuid"></param>
+        /// <returns>WorldObject</returns>
```

Comment:
these param and returns tags are useless.  remove them unless they provide actual value.